types/cesium/index.d.ts

```diff
@@ -5486,7 +5568,44 @@ declare namespace Cesium {
             classificationType?: ClassificationType
         });
 
-        destroy(scene: Scene): void;
+        destroy(): void;
+        getGeometryInstanceAttributes(id: any): GeometryInstance;
+        isDestroyed(): boolean;
+        update(): void;
+    }
+
+    class GroundPolylinePrimitive {
+        readonly allowPicking: boolean;
+        readonly asynchronous: boolean;
+        readonly debugShowShadowVolume: boolean;
+        readonly geometryInstances: any[] | GeometryInstance | undefined;
+        readonly interleave: boolean;
+        readonly ready: boolean;
+        readonly readyPromise: Promise<GroundPolylinePrimitive>;
+        readonly releaseGeometryInstances: boolean;
+
+        show: boolean;
+        appearance: Appearance;
+        classificationType: ClassificationType;
+        debugShowBoundingVolume: boolean;
+
+        static initializeTerrainHeights(): Promise<any>;
+        static isSupported(scene: Scene): boolean;
+
+        constructor(options: {
+            geometryInstances?: any[] | GeometryInstance,
```

Comment:
Should this be `GeometryInstance[] | GeometryInstance` like it is for primitives? - 
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/d25d8ff829b3c2467fdb30559f7b8d34bc6b7919/types/cesium/index.d.ts#L3889

Reply:
That's probably correct, I was modeling off of the `GroundPrimitive` but I think both need to be updated to that.